packages/openmrs/src/http.js

```diff
@@ -42,3 +42,91 @@ export function request(method, path, options = {}, callback = s => s) {
     return util.prepareNextState(state, response, callback);
   };
 }
+
+/**
+ * Make a GET request to any OpenMRS endpoint
+ * @public
+ * @function
+ * @example
+ * get(
+ *  "/ws/rest/v1/patient",
+ *  {
+ *    query: {
+ *      limit: 1
+ *    }
+ *  }
+ *  )
+ * @param {string} path - path to resource
+ * @param {OpenMRSOptions} [options={}] - An object containing query params, headers and body for the request
+ * @returns {operation}
+ */
+export function get(path, options = {}, callback = s => s) {
+  return async state => {
+    const [resolvedPath, resolvedOptions] = expandReferences(state, path, options);
+
+    const response = await util.request(state, 'GET', resolvedPath, resolvedOptions);
+
+    return util.prepareNextState(state, response, callback);
+  };
+}
+
+
+/**
+ * Make a POST  request to an OpenMRS endpoint
+ * @public
+ * @function
+ * @example
+ * post(
+ *  "/ws/rest/v1/patient",
+ *  {
+ *    data: {
+ *      "person": {
+ *      "gender":"M",
+ *      "age":47,
+ *      "birthdate":"1970-01-01T00:00:00.000+0100",
+ *      "names":[
+ *        {
+ *          "givenName":"Jon",
+ *          "familyName":"Snow"
+ *        }
+ *      ],
+ *    }
+ *    }
+ *  }
+ * )
+ * @param {string} path - path to resource
+ * @param {OpenMRSOptions} [options={}] - An object containing query params, headers and body for the request
+ * @returns {operation}
+ */
+export function post(path, options = {}, callback = s => s) {
```

Comment:
Can we change this signature to `post(path, data, options)`?

That simplifies the example significantly. Most users should be able to just provide a path and some JSON data without having to fiddle about with options.